aiidalab/__main__.py

```diff
@@ -560,7 +585,7 @@ def uninstall(app_name, yes, dry_run, force, fully_remove):
 
 
 @contextmanager
-def _mock_schemas_endpoints():
+def _mock_schemas_endpoints() -> Generator[None, None, None]:
```

Comment:
```suggestion
def _mock_schemas_endpoints() -> Iterator[None]:
```

I think this is simpler?

Reply:
Hmm, I guess you're right, and mypy accepts this. However, since we use the Generator type in many other places now, I'd like to keep things consistent for now. Later if we like we can go through and simplify this accross the code base.

Reply:
sure!